### ISSUE TITLE
Can't figure out how to work with closet in the cloze reveal section

### ISSUE THREAD

**Issue by glitchlover** [hgiesel/closet#51]
**Describe the bug**
The front card is working fine, but my problem is in the backcard where I can see [[c1::abc]] instead of just "abc"
![image](https://user-images.githubusercontent.com/59877631/104797022-41eb5080-57e5-11eb-8e3f-6c325279dae4.png)
![image](https://user-images.githubusercontent.com/59877631/104797023-43b51400-57e5-11eb-9e49-b14a8fce9b08.png)

 - AnkiDesktop
 - Windows
 - 2.1.35

**Debug information**

#### Method 1
here is the web inspecter of the card
![image](https://user-images.githubusercontent.com/59877631/104796981-f638a700-57e4-11eb-807a-3996abf997d5.png)
And here is the console log
![image](https://user-images.githubusercontent.com/59877631/104796987-03559600-57e5-11eb-921f-251697be3fa7.png)

# Method 2
here is my anki card
[Default__Computer Science.zip](https://github.com/hgiesel/closet/files/5823832/Default__Computer.Science.zip)

Thank you for your patient

**Comment by Michael35699** [comment]
Support thread in anki forums described the fix for this issue:

https://forums.ankiweb.net/t/closet-for-anki-official-support/4560/11

TL;DR: Update your anki version
